src/components/Gifs/Gifs.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import PropTypes from "prop-types";
+
+// actions
+import { addFavorites, getFavorites, deleteFavorites } from "./../../actions";
+
+// UI
+import UI from './Gifs.ui';
+
+class Search extends Component {
```

Comment:
I'm not sure why are you using two components to map one container. You could just use one component and one container.